tools/generate_hotkeys.py

```diff
@@ -9,11 +10,56 @@
 OUTPUT_FILE = Path(__file__).resolve().parent.parent / "docs" / "hotkeys.md"
 SCRIPT_NAME = PurePath(__file__).name
 
+# Exclude keys from duplicate keys checking
+KEYS_TO_EXCLUDE = ["q", "e", "m", "r"]
+
 
 def main() -> None:
     generate_hotkeys_file()
 
 
+def lint_hotkeys_file() -> None:
+    """
+    First lint config/keys.py file for key description and key duplication, then compare if in sync with existing hotkeys.md file.
+    """
+    hotkeys_file_string = get_hotkeys_file_string()
+    # To lint keys description and duplicate keys
+    error_flag = 0
+    help_text_style = re.compile(r"^[^\d.]*$")
+    categories = read_help_categories()
+    for action in HELP_CATEGORIES.keys():
+        check_duplicate_keys_list: List[str] = []
+        for help_text, key_combinations_list in categories[action]:
+            check_duplicate_keys_list.extend(key_combinations_list)
+            various_key_combinations = various_key_combination(key_combinations_list)
+            # Check description style
+            if not re.match(help_text_style, help_text):
+                print(
+                    f"Description - ({help_text}) for key combination - [{various_key_combinations}] should contain only alphabets, spaces and special characters except ."
+                )
+                error_flag = 1
+        # Check key combination duplication
+        check_duplicate_keys_list = [
```

Comment:
You can probably split this commit into 2. One for checking description style and the other for checking key combination duplication.